CloudyCider/views/components/EC2/EC2InstanceView.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  EC2InstanceView.swift
+//  CloudyCider
+//
+//  Created by Matvii Hodovaniuk on 7/10/19.
+//  Copyright © 2019 Matvii Hodovaniuk. All rights reserved.
+//
+
+import SwiftUI
+
+struct EC2InstanceView: View {
```

Comment:
this file uses 4 spaces for formatting, while the rest of the project uses 2 spaces